tests/admin_views/tests.py

```diff
@@ -368,15 +368,25 @@ def test_change_list_sorting_callable_query_expression(self):
         Query expressions may be used for admin_order_field. (column 9 is
         order_by_expression in ArticleAdmin).
         """
-        response = self.client.get(reverse('admin:admin_views_article_changelist'), {'o': '9'})
-        self.assertContentBefore(
-            response, 'Oldest content', 'Middle content',
-            'Results of sorting on callable are out of order.'
-        )
-        self.assertContentBefore(
-            response, 'Middle content', 'Newest content',
-            'Results of sorting on callable are out of order.'
-        )
+        tests = (  # (description, index)
+            ('F("date") + DurationValue(3 days, 0:00:00)', 9),
+            ('F("date")', 12),
+            ('F("date").asc(nulls_last=True)', 13),
```

Comment:
I think that property name i.e. `order_by_expression`, `order_by_f_expression`, or `order_by_orderby_expression` is a sufficient description.